src/Compilers/Core/Portable/Syntax/SyntaxToken.cs

```diff
@@ -22,6 +22,7 @@ public struct SyntaxToken : IEquatable<SyntaxToken>
     {
         internal static readonly Func<SyntaxToken, bool> NonZeroWidth = t => t.Width > 0;
         internal static readonly Func<SyntaxToken, bool> Any = t => true;
+        internal static readonly Func<DiagnosticInfo, Diagnostic> CreateDiagnosticWithoutLocation = Diagnostic.Create;
```

Comment:
>internal [](start = 8, length = 8)

Should we use ```private``` accessibility? Do we expect other types to take advantage of this field?